utils/maintenance/Release_Notes_Command.php

```diff
@@ -149,11 +163,14 @@ private function get_bundle_release_notes( $source, $format ) {
 			// Closed milestones denote a tagged release
 			$milestones = GitHub::get_project_milestones(
 				$package_name,
-				array( 'state' => 'closed' )
+				[ 'state' => 'closed' ]
 			);
 			foreach ( $milestones as $milestone ) {
-				if ( ! version_compare( $milestone->title, $version_constraint,
-					'>' ) ) {
+				if ( ! version_compare(
```

Comment:
Any particular reason for this (not all that long) function call to be multi-line ?